### ISSUE TITLE
Accordions lack sufficient color contrast if used with background colors

### ISSUE THREAD

**Issue by bspeare** [uiowa/uids#665]
# Problem 

Accordions lack sufficient color contrast if used with background colors

# Proposed Solution

- Update template and `accordion.scss` to work with background colors
